component/GroceriesList/GroceriesList.styled.js

```diff
@@ -1,20 +1,24 @@
 import styled from "styled-components";
+import Link from "next/link";
 
-export const StyledList = styled.section`
+export const StyledList = styled.ul`
+  list-style-type: none;
   padding: 0.5rem;
   gap: 5px;
```

Comment:
`gap` has no effect on this element since it’s not a flex or a grid container. Therefore you could remove this property.
```suggestion
```